src/coreclr/jit/targetarm64.h

```diff
@@ -52,6 +52,8 @@
   #define LAST_FP_ARGREG           REG_V15
   #define REG_PREDICATE_FIRST      REG_P0
   #define REG_PREDICATE_LAST       REG_P15
+  #define REG_PREDICATE_LOW_FIRST  REG_P0
```

Comment:
nit: should we just use REG_PREDICATE_FIRST instead? Yes, I appreciate the symmetry.

Reply:
I went back and too over this when writing. Removed.